ci: Agama sonar check is failing

**Describe the bug**
Sonar scanning of `Agama` module is failing for #1800 

**To Reproduce**
Steps to reproduce the behavior:
1. Create a PR with changes in Agama code
2. Let the PR checks run
3. Agama check fails with following error

**Expected behavior**
Agama build should work and Sonar check should be able to connect to remote server.

**Screenshots**

```
Error:  Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.9.1.2184:sonar (default-cli) on project agama: Unable to execute SonarScanner analysis: Fail to get bootstrap index from server: Failed to connect to localhost/0:0:0:0:0:0:0:1:9000: Connection refused (Connection refused) -> [Help 1]
Error: 
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error: 
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Error: Process completed with exit code 1.
```

--- a/agama/pom.xml
+++ b/agama/pom.xml
@@ -25,6 +25,11 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.min-version>3.6.3</maven.min-version>
         <jans.version>${project.version}</jans.version>
+
+        <sonar.projectKey>JanssenProject_jans-agama</sonar.projectKey>
+        <sonar.moduleKey>${project.groupId}:${project.artifactId}</sonar.moduleKey>
+        <sonar.organization>janssenproject</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
     </properties>
 
     <repositories>

--- a/agama/pom.xml
+++ b/agama/pom.xml
@@ -26,7 +26,7 @@
         <maven.min-version>3.6.3</maven.min-version>
         <jans.version>${project.version}</jans.version>
 
-        <sonar.projectKey>JanssenProject_jans-agama</sonar.projectKey>
+        <sonar.projectKey>janssenproject_jans-agama</sonar.projectKey>
         <sonar.moduleKey>${project.groupId}:${project.artifactId}</sonar.moduleKey>
         <sonar.organization>janssenproject</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>

